Symfony/CS/AbstractAlignFixer.php

```diff
@@ -86,4 +91,21 @@ protected function replacePlaceholder(Tokens $tokens, $deepestLevel)
 
         return $tmpCode;
     }
+
+    /**
+     * Cleans up extra line between variable and placeholder.
+     * Ex: `$ccc  = 1` becomes `$ccc = 1`.
+     *
+     * @param string $line
+     * @param string $placeholder
+     *
+     * @return string
+     */
+    private function cleanInnerSpaces($line, $placeholder)
```

Comment:
```php
$a = 1;
$bbb      = 2;
```

In original code we have operations:
1. put 7 spaces after `$a`.
That is all, quick and fast.

In proposed code we have operations:
1. ensure there is only one space after `$a`
2. ensure there is only one space after `$b`
3. add 2 spaces after first `=`
4. add 0 (:D) spaces after second `=`

operations 1 and 2 run for every `=` and we don't want them to waste our time.

Why not just:
1. add 2 spaces after `$a`
2. trim 5 spaces before `$b`
?

-----------------------------

now, you are removing spaces and add spaces instead of adding less spaces